gateway-service/src/main/java/org/zowe/apiml/gateway/filters/post/DebugHeaderFilter.java

```diff
@@ -49,7 +49,9 @@ public boolean shouldFilter() {
     @Override
     public Object run() {
         String debug = convertToPrettyPrintString(Debug.getRoutingDebug());
-        log.debug("Filter Debug Info = \n{}", debug);
+        String reqInfo = RequestContext.getCurrentContext().getFilterExecutionSummary().toString();
+        log.debug("Filter Execution Summary: " + reqInfo);
+        log.debug("RoutingDebug: " + debug);
```

Comment:
Hi Tim, when i try to run your PR, i do not see the messages you added in log. Can you see them? Is there another setup that needs to be done for this to start working?

Reply:
a user in Zowe would enable debug like this: https://docs.zowe.org/stable/troubleshoot/troubleshoot-apiml.html#enable-api-ml-debug-mode

Reply:
which results in setting `spring.profiles.include=debug`.

Reply:
I retested and i must have made an error. I sincerely appologize.